database/migrations/2020_10_18_200146_set_collegist_status.php

```diff
@@ -14,7 +14,9 @@ class SetCollegistStatus extends Migration
      */
     public function up()
     {
-        $collegists = User::collegists();
+        // we have to obtain these in a legacy way,
+        // as we do not have 'valid_from' and 'valid_until' columns yet
+        $collegists = User::withRole(Role::collegist(), includesExpired: true);
```

Comment:
This looks very troubling to me.

Are we trying to apply the current models on a database whose schema is quite different from what we use now? Should we maybe stop doing that and just interact with the database directly? IIRC we had already run into issues with migrations when deploying the previous update.

Reply:
You are right; I'll change that:)

Reply:
Actually, it turned out that this migration does not do nothing:D it does not even once enter the foreach loop, where it would call a non-existent function. So I simply deleted it.